### DISCUSSION TITLE
GOS for parent reconstructed grains

### DISCUSSION THREAD

**Discussion by mugikumar** [ORTools4MTEX/ORTools#31] [General]
Hi 
   I have been trying to map the GOS for parent reconstructed grains. i couldn't get the map. please refer the TSL map of GOS below. please help me to get the code
![GOS for parent](https://user-images.githubusercontent.com/109027461/216907935-552302ba-ec25-48bf-b8fd-95e8be1d1272.jpg)

**Comment by frankNiessen** [comment]
Hi Mugikumar, 

I can only highly advice against determining the GOS from reconstructed parent grain data. You cannot get the GOS from the reconstructed grains, which only have an approximate mean orientation. If you apply it to the reconstructed parent EBSD data there will be local misorientations that are associated with local variations in the orientation relationship and possible errors associated with parent grain reconstruction. When computing the GOS you will falsely interpret these local misorientations as GOS. This will produce absolut rubbish, so please DO NOT DO IT =)

Best wishes
Frank

**Reply by mugikumar** [reply]
Thank you Frank for your clarification.
